helper.sh

```diff
@@ -220,11 +194,12 @@ if [[ "$#" -ge 1 ]]; then
             --docker) docker=true; ((command_num++)); shift ;;
             -v) verbose=true; shift ;;
 
-            --) shift ; arguments+=$@; break ;;
+            --) shift ; arguments+=("${@}"); break ;;
```

Comment:
one of the reasons to hate bash

Reply:
we've completely ditched it and replaced it with Python, it may be a good idea to do it as well

Reply:
(somewhere in the future, this is just a food for thought)

Reply:
agree, i was thinking about grunt (js). It is basically a replacement for make and for the helper script at once

Reply:
I'd rather go for gulp then, everybody uses it now

Reply:
> Each plugin for Gulp is written with the goal of doing one thing very well.

Unix philosophy? I'm in

Reply:
@WikiRik can you write it off as a low priority task?

Reply:
I've also used gulp before webpack (they are slightly different though: gulp is a task manager when webpack is a frontend build tool), worked flawlessly for me

Reply:
> @WikiRik can you write it off as a low priority task?

What exactly?